guide/src/guide/new-cell.md

```diff
@@ -1 +1,221 @@
-# Write a New Cell (TODO)
+# Write a New Cell
+
+Following this guide, you can write a custom cell yourself.
+
+## Basic Structure
```

Comment:
It is important to convey the core concepts of a cell to the developers reading this guide. What is a cell? How is it composed? Which traits should be implemented, and what logic should be implemented in what methods?

We should first state these high-level pictures and then on the details of how to implement a illustrating cell (i.e., LossCell)